mobileversion.html

```diff
@@ -12,10 +12,21 @@
      <title>Mobile Version Website</title>
   </head>
     <body>
+      <section class="mobileversion">
        <header class="header">
-        <nav>
-     <a href="#">My Logo</a>
-     <a href="#"><i class="fa fa-bars"></i></a>
+        <nav class="nav-bar">
+     <a class="log" href="#">My Logo</a>
+    <ul class="nav-menu">
+    
+    <li class="nav-item"><a href="#Portfolio" class="nav-link">Portfolio</a></li>
+    <li class="nav-item"><a href="#About" class="nav-link">About</a></li>
+    <li class="nav-item"><a href="#Contact" class="nav-link">Contact</a></li>
+    </ul>
+    <div class="hamburger">
+    <span class="bar"></span>
+    <span class="bar"></span>
+    <span class="bar"></span>
+    </div>
```

Comment:
### Your Mobile Menu
![Capture](https://user-images.githubusercontent.com/93039359/201867995-c8d81a57-e664-46cf-a50e-56ee5b130a0c.PNG)
### Figma Mobile Menu
![figma](https://user-images.githubusercontent.com/93039359/201868099-20353719-271d-4ca7-be04-c6c596942afd.PNG)


- You can see that your mobile menu is not properly following the Figma template.
- [ ] First thing is that you can see a white space at the top of the screen. Kindly remove it 😊
- [ ] Second thing is that your close icon is not the same color as the Figma template 😨 Kindly fix it 👍
- [ ] Third thing is that you can see that at the end of your mobile menu you can see a black line in the Figma template. Kindly add that black border line in your design at the end of your mobile menu 👍
- [ ] Last thing is that your mobile menu should have a blur effect on the background means everthing behind your mobile menu should be blur to follow the Figma template. Kindly fix it 👍